ahnlich/ai/src/manager/mod.rs

```diff
@@ -197,8 +191,8 @@ impl ModelManager {
         inputs: Vec<StoreInput>,
         preprocess_action: PreprocessAction,
     ) -> Result<Vec<StoreKey>, AIProxyError> {
-        let supported = model.into();
-        if let Some(sender) = self.models.get(&supported) {
+        let model = model.into();
```

Comment:
We wouldn't need `AIModel` to convert into `Model` anymore, it can just convert to `SupportedModel`, and that would be used to retrieve the model thread mpsc sender that we need